e2e/run-e2e.sh

```diff
@@ -0,0 +1,145 @@
+#!/usr/bin/env bash
+
+# variables
+KIBANA_PORT=5601
+ELASTICSEARCH_PORT=9201
+
+# ensure Docker is running
+docker ps &> /dev/null
+if [ $? -ne 0 ]; then
+    echo "⚠️  Please start Docker"
+    exit 1
+fi
+
+# formatting
+bold=$(tput bold)
+normal=$(tput sgr0)
+
+# paths
+E2E_DIR="${0%/*}"
+TMP_DIR="tmp"
+APM_IT_DIR="tmp/apm-integration-testing"
+DOCKER_DIR="../examples/docker/"
+
+cd ${E2E_DIR}
+
+KIBANA_VERSION=7.10.0
+
+#
+# Create tmp folder
+##################################################
+echo "" # newline
+echo "${bold}Temporary folder${normal}"
+echo "Temporary files will be stored in: ${E2E_DIR}${TMP_DIR}"
+mkdir -p ${TMP_DIR}
+
+
+# Start synthetics docker examples
+echo "" # newline
+echo "${bold}Starting synthetics docker examples${normal}"
+echo "" # newline
+cd ${DOCKER_DIR}
+
+./run.sh 7.10.0 \
```

Comment:
better to just pass it to the `run.sh` script instead of handling the version and heartbeat args on both places.